### PR TITLE
A Intuit sign-up website clone - Built with HTML & CSS 

### PR DESCRIPTION
**Pull request by idgm5**
We recreated the Intuit sign-up web page following the instructions required by [The Odin Project](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients) and [Microverse Curriculum ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/).

Any new changes will be documented and pushed on this pull request.

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!

Here are changes you need to make before this project can be approved:
* [x] The intuit logo should be an embedded image.([link to page](https://accounts.intuit.com/signup.html)) Right now you are adding it as a background, but you should add it as an `img`. 
* [x] Product's row should contain 3 images (just like main logo above)  and text (logo on the left, text on the right).
* [x] Your project is missing disclaimer text from original website, you can find it under phone input:
image
* [x] You need to have only one form in this project, no need to put each input in separate `form` tag.
I suggest for you to change `Forms` div to form and all forms you have inside to div's.
* [x] Phone `<input>` should be of type `tel` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
* [x] `Footbar` div should be inside a footer tag, or just change div to footer.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team,  You have done a great job with this project. :clap:, but there is just one more thing you need to change, check inline comments.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                                <input type="email" required>
+                            </div>
+                            <div>
+                              <label>Phone (recommended)</label>
+                              <input type="tel">
+                            </div>
+                          <div class="lockPassword">
+                              <label>Password</label>
+                              <input type="password" placeholder="&#xf13e;   ">
+                              <p>Standard call, messaging or data rates may apply.</p>
```
Comment (kobiyoyo) [inline index.html]: This should be under Phone input not password

**Pull request merged**